tests/LoginScreen.spec.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import { sessionService } from 'redux-react-native-session';
+import Config from 'react-native-config';
+import nock from 'nock';
+
+import LoginScreen from 'containers/LoginScreen';
+import configureStore from 'store/configureStore';
+
+describe('<LoginScreen />', () => {
+  const userEmail = 'example@example.com';
+  const userPassword = 'example';
+  let wrapper;
+  let emailInput;
+  let passwordInput;
+  let submitButton;
+  let response;
+  let data;
+
+  beforeEach(() => {
+    const store = configureStore();
+    wrapper = mount(
+      <Provider store={store}>
+        <LoginScreen />
+      </Provider>
+    );
+
+    data = {
+      user: {
+        email: userEmail,
+        password: userPassword,
+      }
+    };
+
+    response = {
+      user: {
+        username: null,
+        password: null,
+        id: 1,
+        email: userEmail,
+        firstName: null,
+        lastName: null,
+        clientPersonCount: 0,
+        countryCode: null,
+        clientPersons: []
```

Comment:
I think you can remove some values like: `clientPersonCount` and `countryCode` just leave `username`, `password` `email` or the ones you need and are generic. The rest depends on the project.